pyproject.toml

```diff
@@ -35,18 +35,19 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.6,<3.7"
-boto3 = "^1.10.46"
-botocore = "^1.13.46"
-certifi = ">=2020.4.5.2"
+python = ">=3.6.1,<3.7"
+boto3 = "^1.10.50"
+botocore = "^1.13.50"
+certifi = ">=2020.11.8"
 chardet = "3.0.4"
 # TODO: These next three are not used in cgap. Are they really needed? -kmp 29-Apr-2020
-"collective.recipe.cmd" = "0.11"
-"collective.recipe.modwsgi" = "2.1"
-"collective.recipe.template" = "1.13"
+# I think .recipe is a buildout thing so I'm going to try commenting this out as a start. Delete later. -kmp 15-Dec-2020
+# "collective.recipe.cmd" = "0.11"
+# "collective.recipe.modwsgi" = "2.1"
+# "collective.recipe.template" = "1.13"
 colorama = "0.3.3"
-dcicsnovault = "^4.0.12"
-dcicutils = "^1.3.0"
+dcicsnovault = "4.1.0"
+dcicutils = "^1.4.0"
```

Comment:
Is there a reason we aren't aligning to a higher version? Is it because of the library update for `elasticsearch`? It should be ok to bump that as well.

Reply:
At this point I'm just trying to match what we did in CGAP, which had this pinned. It seemed unlikely that Fourfront needs a newer version than CGAP.